source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/show/display.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { ProjectResource, attributesFor } from '@data';
+import { i18nProps } from '@lib/i18n';
+
+
+
+import { IExpectedPropsForRoute } from './types';
+import Header from './header';
+import Details from './details';
+
+interface IExpectedProps {
+  project: ProjectResource;
+}
+
+type IProps =
+  & IExpectedPropsForRoute
+  & IExpectedProps
+  & i18nProps;
+
+export default class Display extends React.Component<IProps> {
+  render() {
+    const { project } = this.props;
+
+    return (
+      <div className='ui container'>
+        <div className='project-details' data-test-project>
```

Comment:
can we refactor project-details into our one of our utilities?

Reply:
also data-test-project is same name as in project view

Reply:
why?
also, the data-test-project being the same anywhere doesn't matter. :)